controls/expander/getting-started.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Getting Started
+page_title: Getting Started with Xamarin.Forms Expander Control
+description: "Check our Getting Started documentation article for Telerik Expander for Xamarin control."
+position: 1
+slug: expander-getting-started
+---
+
+# .NET MAUI Expander Getting Started
```

Comment:
follow the getting started guide as other telerik .net maui control's article. This stucture is from xamarin getting started, use maui getting started